### ISSUE TITLE
8-19 Spain UA yields on conquest

### ISSUE THREAD

**Issue by randomnublet** [LoneGazebo/Community-Patch-DLL#8213]
Gold/faith on conquest, based on # city tiles needs to be looked at. 

3 examples, same Era (Medieval).

![civspainuabug5](https://user-images.githubusercontent.com/53142313/131217483-3b7ab06f-dfb6-482b-8d65-4bbfd429d768.png)

This is the pretty lame city of Dortmund, it has been founded recently and is only 7 tiles wide. It's going to fall 2 turns later for a whooping 1660 gold, 664 faith.

![civspainuabug3](https://user-images.githubusercontent.com/53142313/131217552-de09597b-8e1b-42b7-8c25-47c33c9fd3cf.png)
![civspainuabug4](https://user-images.githubusercontent.com/53142313/131217557-686237c0-9319-444a-8ebd-e30afd509f52.png)

Hamburg on the other hand has already claimed 16 tiles, yet it's only worth 1620 gold 648 faith

![civspainuabug2](https://user-images.githubusercontent.com/53142313/131217595-21480a54-ceeb-441c-985d-8b857a0158d0.png)
![civspainuabug1](https://user-images.githubusercontent.com/53142313/131217597-fc80bb1a-51f1-44c4-8ea4-b3704971af66.png)

Taking over Berlin, 24 tiles: 480 gold, 192 faith. This is the only number that makes senses, since the UA claims it should net you 10 gold 4 faith per tile, scaling with era (20/8 in Medi). I remember yields on conquest being higher than those from peaceful settling or tile aquisition though.

Hence I have the feeling the UA doesn't determine properly which tiles belong to each city, if the city is adjacent to another one then all tiles from all cities are considered (in the Dortmund example, it net me 20 gold 8 faith per 83 times, that's the whole extension of Germany as you can see from this last screenshot (I didn't save at the exact turn, but Munich claimed that wheat and became adjacent to Frankfurt)

![civspainuabuglast](https://user-images.githubusercontent.com/53142313/131217977-e11d4006-f811-4939-b2d5-dc570f0e3027.png)

With the last wheat that's 83 contiguous tiles.

tl;dr 

Spain UA considers all contiguous tiles of a civ as if they belong to the city you've just conquered. Island metropolis or the last capital standing are worth less than a freshly founded city if its borders touch the rest of the (yet to be conquered) empire.
